phpunit/functional/RuleTicketTest.php

```diff
@@ -3652,4 +3653,101 @@ public function testWriterCriterion()
         $this->assertEquals($user_id, (int)$ticket->getField('users_id_recipient'));
         $this->assertEquals($requesttypes_id, (int)$ticket->getField('requesttypes_id'));
     }
+
+    public function testDoNotComputeStatusFollowupWithRule()
+    {
+        $this->login('glpi', 'glpi');
+
+        $followuptemplate = new \ITILFollowupTemplate();
+        $templateid = $followuptemplate->add($templateinput = [
+            'name' => 'followuptemplate_' . __FUNCTION__,
+            'content' => 'Test',
+        ]);
+        $this->checkInput($followuptemplate, $templateid, $templateinput);
+
+        $ruleticket = new \RuleTicket();
+        $rulecrit   = new \RuleCriteria();
+        $ruleaction = new \RuleAction();
+
+        $ruletid = $ruleticket->add($ruletinput = [
+            'name'         => 'test do not compute status followup',
+            'match'        => 'AND',
+            'is_active'    => 1,
+            'sub_type'     => 'RuleTicket',
+            'condition'    => \RuleTicket::ONADD,
+            'is_recursive' => 1,
+        ]);
+        $this->checkInput($ruleticket, $ruletid, $ruletinput);
+
+        $crit_id = $rulecrit->add($crit_input = [
+            'rules_id'  => $ruletid,
+            'criteria'  => 'name',
+            'condition' => \Rule::PATTERN_CONTAIN,
+            'pattern'   => 'test',
+        ]);
+        $this->checkInput($rulecrit, $crit_id, $crit_input);
+
+        $act_id = $ruleaction->add($act_input = [
+            'rules_id'    => $ruletid,
+            'action_type' => 'assign',
+            'field'       => 'status',
+            'value'       => CommonITILObject::WAITING,
+        ]);
+        $this->checkInput($ruleaction, $act_id, $act_input);
+
+        $act2_id = $ruleaction->add($act2_input = [
+            'rules_id'    => $ruletid,
+            'action_type' => 'append',
+            'field'       => 'itilfollowup_template',
+            'value'       => $templateid,
+        ]);
+        $this->checkInput($ruleaction, $act2_id, $act2_input);
+
+        $user1 = new \User();
+        $user1->getFromDBbyName('glpi');
+        $this->assertGreaterThan(0, $user1->getID());
+
+        $user2 = new \User();
+        $user2->getFromDBbyName('tech');
+        $this->assertGreaterThan(0, $user2->getID());
+
+        $ticket = new \Ticket();
+        // Assigning technician after ticket is already closed should be blocked
```

Comment:
It's good to comment, but I'm not sure if this one corresponds to the following code?